### PR TITLE
docs: add login ui customization tutorial

### PR DESCRIPTION
**Pull request by leo220yuyaodog**
close: #375 

Signed-off-by: magicwind <2814461814@qq.com>

### CODE REVIEW

**Comment by leo220yuyaodog** [comment]
@Selflocking @yoyo314 Can you help me with review? Thanks.

File: docs/application/ui-customization.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+:::
+
+After fill the `form CSS`, don't forget to save the config at the bottom. Ok, let's see the effect.
+
+![step2_end.png](/img/application/ui-customization/step2_end.png)
+
+## Part3: Panel position
+
+Now the login page is much prettier than the beginning.  We also provider three buttons for you to decide the position of the panel.
```
Comment (Selflocking) [inline docs/application/ui-customization.md:58]:
```suggestion
Now the login page is much prettier than the beginning. We also provide three buttons for you to decide the position of the panel.
```

File: docs/application/ui-customization.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+More introduction about application in Casdoor:
+
+- [Signup Items Table](/docs/application/signup-items-tabel)
+- [Application Config](/docs/application/config)
+
+Thanks for reading!
+
+
+
+
```
Comment (Selflocking) [inline docs/application/ui-customization.md:161]: plz leave only one blank line.

File: docs/application/ui-customization.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+![step3_position.png](/img/application/ui-customization/step3_position.png)
+
+For example, select the **Right** button:
+
+![step3_end.png](/img/application/ui-customization/step3_end.png)
+
+## Part4: Enable the side panel
+
+You will see now how you can enable a side panel and custom the style.
```
Comment (Selflocking) [inline docs/application/ui-customization.md:68]:
```suggestion
You will see now how you can enable a side panel and custom the style.
```

File: docs/application/ui-customization.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+## Part4: Enable the side panel
+
+You will see now how you can enable a side panel and custom the style. 
+
+First, select the button. In **enable side panel** mode, the panel will in center.
+
+![enable_side_panel.png](/img/application/ui-customization/step4_enable_side_panel.png)
+
+Then edit the `Side panel HTML`, it decides what content will show in the side panel. Same as the `Form CSS`, we provide a default template.
+Just copy and paste.
```
Comment (Selflocking) [inline docs/application/ui-customization.md:75]:
```suggestion
Just copy and paste.
```

File: docs/application/ui-customization.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+<div class="left-model">
+  <span class="side-logo"> <img src="https://cdn.casbin.org/img/casdoor-logo_1185x256.png" alt="Casdoor" style="width: 120px"> 
+    <span>SSO</span> 
+  </span>
+  <div class="img">
+    <img src="https://cdn.casbin.org/img/casbin.svg" alt="Casdoor"/>
+  </div>
+</div>
+```
+Let's see the effect. The side panel with a logo and image is shown, but the result didn't see well.
```
Comment (Selflocking) [inline docs/application/ui-customization.md:114]:
```suggestion
Let's see the effect. The side panel with a logo and image is shown, but the result didn't see well.
```

File: docs/application/ui-customization.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+```
+Let's see the effect. The side panel with a logo and image is shown, but the result didn't see well. 
+
+![step4_result1.png](/img/application/ui-customization/step4_result1.png)
+
+You need to modify and add some css in `form CSS`.
+
+![step4_modify_CSS.gif](/img/application/ui-customization/step4_modify_CSS.gif)
+
+The final code is as follows.
```
Comment (Selflocking) [inline docs/application/ui-customization.md:122]:
```suggestion
The final code is as follows.
```

**Review by Selflocking — CHANGES_REQUESTED** [review]

**Review by Selflocking — CHANGES_REQUESTED** [review]

**Pull request merged**